ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -64,9 +64,19 @@ azure_connection::put_object(const std::string &file_name) const
     return 0;
 }
 
+// Delete an object in the bucket given the object name.
 int
-azure_connection::delete_object() const
+azure_connection::delete_object(const std::string &object_name) const
 {
+    std::string obj = _object_prefix + object_name;
+
+    auto object_client = _azure_client.GetBlobClient(obj);
+    auto delete_blob_response = object_client.DeleteIfExists();
+
+    // Returns false if obj doesn't exist.
+    if (!delete_blob_response.Value.Deleted)
```

Comment:
add a message if the object doesn't exist to std::cerr

Reply:
done :)